### PR TITLE
Migrate weights files from webnn-samples

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, PTAL, thanks!

File: models/style_transfer/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+`.npy` files under this folder are downloaded from models of [Fast Style Transfer](https://github.com/acerwebai/VangoghCrazyWorld) from Vangogh Crazy World project, which is licensed under the MIT.
```
Comment (huningxin) [inline models/style_transfer/README.md]: How about aligning the folder name as `fast_style_transfer_nchw`?

File: models/lenet/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+`lenet.bin` file under this folder is copied from [the LeNet example of OpenVINO*](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/samples/ngraph_function_creation_sample/), which is licensed under the Apache 2.0.
```
Comment (huningxin) [inline models/lenet/README.md]: how about aligning the folder name as `lenet_nchw`?

File: models/squeezenet_nchw/README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+`.npy` files under this folder are downloaded from model of [SqueezeNet 1.1](https://github.com/onnx/models/tree/master/vision/classification/squeezenet),
```
Comment (huningxin) [inline models/squeezenet_nchw/README.md]: folder name `squeezenet1.1_nchw`?

File: models/nsnet2/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+`.npy` files under this folder are downloaded from model of [NSNet2](https://github.com/microsoft/DNS-Challenge/blob/master/NSNet2-baseline/nsnet2-20ms-baseline.onnx), which is licensed under the MIT.
```
Comment (huningxin) [inline models/nsnet2/README.md]: folder name `nsnet2_nchw`?

File: models/squeezenet_nhwc/README.md [hunk 5]
```diff
@@ -0,0 +1 @@
+`.npy` files under this folder are downloaded from model of [SqueezeNet 1.0](https://storage.googleapis.com/download.tensorflow.org/models/tflite/model_zoo/upload_20180427/squeezenet_2018_04_27.tgz), which is licensed under the Apache 2.0.
```
Comment (huningxin) [inline models/squeezenet_nhwc/README.md]: folder name `squeezenet1.0_nhwc`

**Review by huningxin — COMMENTED** [review]
Thanks @Honry , some ideas about folder naming. please take a look.

**Comment by Honry** [comment]
@huningxin, done. :)

**Review by huningxin — APPROVED** [review]
LGTM. Thanks @Honry

**Comment by huningxin** [comment]
@Honry , do you want to add a github workflow to auto deploy it to github pages?

**Comment by Honry** [comment]
@huningxin，how about build GitHub Pages from `master` branch directly? As this repo only includes test data.

**Comment by huningxin** [comment]
> @huningxin，how about build GitHub Pages from `master` branch directly? As this repo only includes test data.

Sounds good to me, let me merge and setup the branch. Thanks for the suggestion.

**Comment by huningxin** [comment]
@Honry , done, please check it out:

https://webmachinelearning.github.io/test-data/

**Pull request merged**